pyaedt/application/Analysis.py

```diff
@@ -408,6 +408,174 @@ def SolutionTypes(self):
+                        exported_files.append(szExportPath)
+                    if self.solution_type in ["HFSS3DLayout", "HFSS 3D Layout Design", "HFSS", "Circuit"]:
+                        try:
+                            szExportPath = os.path.join(export_folder,
+                                                        "{0}.s{1}p".format(self.project_name, excitations))
+                            self.osolution.ExportNetworkData(variation_array[0], ["{0}:{1}".format(s, sweep)], 3,
+                                                             szExportPath, ["All"], True, 50, "S", -1, 0, 15, True,
+                                                             False,
+                                                             False)
+                            exported_files.append(szExportPath)
+                            self.logger.info("Export Touchstone: %s", szExportPath)
+                        except:
+                            self.logger.debug("Export SnP Failed: No Solution")
+
+                else:
+                    varCount = 0
+                    for variation in variation_array:
+                        varCount += 1
+                        szExportPath = os.path.join(export_folder, "{0}_{1}.prof".format(self.project_name, varCount))
+                        result = self.export_profile(s, variation, szExportPath)
+                        if result:
+                            exported_files.append(szExportPath)
+                        szExportPath = os.path.join(export_folder, "{0}_{1}.conv".format(self.project_name, varCount))
+                        self.logger.info("Export Convergence: %s", szExportPath)
+                        result = self.export_convergence(s, variation, szExportPath)
+                        if result:
+                            exported_files.append(szExportPath)
+                        if self.solution_type in ["HFSS3DLayout", "HFSS 3D Layout Design", "HFSS", "Circuit"]:
+                            try:
+                                szExportPath = os.path.join(export_folder,
+                                                            "{0}_{1}.s{2}p".format(self.project_name, varCount,
+                                                                                   excitations))
+                                self.logger.info("Export SnP: {}".format(szExportPath))
+                                self.osolution.ExportNetworkData(variation, ["{0}:{1}".format(s, sweep)], 3,
+                                                                 szExportPath,
+                                                                 ["All"], True, 50, "S", -1, 0, 15, True, False, False)
+                                exported_files.append(szExportPath)
+                                self.logger.info("Export Touchstone: %s", szExportPath)
+                            except:
+                                self.logger.info("Export SnP Failed: No Solution")
```

Comment:
Should it be a warning or an error message here if the Snp export fails?

Reply:
No, just warning is enough